app/models/api_docs/service.rb

```diff
@@ -1,32 +1,38 @@
 # encoding: utf-8
+
 class ApiDocs::Service < ApplicationRecord
   CURRENT_SWAGGER_VERSION = 2
 
   include SystemName
   extend System::Database::Scopes::IdOrSystemName
 
   belongs_to :account, required: true
-  belongs_to :service, class_name: '::Service', inverse_of: :api_docs_services
+  belongs_to :service, class_name: '::Service', inverse_of: :deprecated_api_docs, required: false
```

Comment:
If we go with https://github.com/3scale/porta/pull/2116/files?file-filters%5B%5D=.rb#r489448808, maybe then we can remove this, and also https://github.com/3scale/porta/blob/e8a21d46d8cd8e54f3c63f2a492d869e9e107861/app/models/service.rb#L23 and https://github.com/3scale/porta/blob/e8a21d46d8cd8e54f3c63f2a492d869e9e107861/app/models/service.rb#L61.